src/status_im2/core.cljs

```diff
@@ -1,9 +1,5 @@
 (ns status-im2.core
   (:require
-    ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
-    [status-im2.setup.i18n-resources :as i18n-resources]
-
-    #_{:clj-kondo/ignore [:unsorted-required-namespaces]}
```

Comment:
@flexsurfer - We should not remove this config. These changes need to be reverted. The translation resources need to be loaded first.

Right now, the app fails to load the translation.

<img width="300" src="https://github.com/status-im/status-mobile/assets/19339952/0486a39a-f282-40e7-b43f-6616207a0d02" />

Reply:
yeah, thanks, I thought we excluded this file